src/taskpane/components/App.tsx

```diff
@@ -4,8 +4,12 @@ import HeroList, { HeroListItem } from "./HeroList";
 import TextInsertion from "./TextInsertion";
 import { makeStyles } from "@fluentui/react-components";
 import { Ribbon24Regular, LockOpen24Regular, DesignIdeas24Regular } from "@fluentui/react-icons";
+import { insertText } from "../taskpane";
+
+/* global Office */
 
 interface AppProps {
+  host: Office.HostType;
```

Comment:
This `host` prop is never used. Can we remove it from the interface?

Reply:
Yes. I have removed it.